src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Double.cs

```diff
@@ -86,7 +87,13 @@ private void WriteNumberValueIndented(double value)
             JsonWriterHelper.WriteIndentation(output.Slice(BytesPending), indent);
             BytesPending += indent;
 
-            bool result = Utf8Formatter.TryFormat(value, output.Slice(BytesPending), out int bytesWritten);
+#if BUILDING_INBOX_LIBRARY
+            var format = new StandardFormat('G');
+#else
+            var format = new StandardFormat('G', 17);
+#endif
+
+            bool result = Utf8Formatter.TryFormat(value, output.Slice(BytesPending), out int bytesWritten, format);
```

Comment:
On .NET Core, if you just use TryFormat without specifying a format, what does it use?

Reply:
StandardFormat returns an empty string, which `double.ToString` then treats as `F`

Reply:
Ok, thanks.